src/components/Onboarding/OnboardingScreen.js

```diff
@@ -0,0 +1,668 @@
+import { makeStyles, createMuiTheme } from '@material-ui/core/styles';
+import Stepper from '@material-ui/core/Stepper';
+import Step from '@material-ui/core/Step';
+import StepLabel from '@material-ui/core/StepLabel';
+import {
+  Button,
+  Typography,
+  Grid,
+  TextField,
+  Container,
+  CardMedia,
+  CardContent,
+  CardActionArea,
+  Card,
+  Box,
+} from '@material-ui/core';
+import { signupUser } from '../../lib/airlock/airlock';
+import Fruit1 from '../../assets/images/Fruit1.svg';
+import Fruit2 from '../../assets/images/Fruit2.svg';
+import Fruit3 from '../../assets/images/Fruit3.svg';
+
+const ROLE_NAMES = ['buyer', 'vendor', 'agency'];
+const theme = createMuiTheme({
+  spacing: 4,
+});
+/* eslint-disable no-debugger, no-console */
+const useStyles = makeStyles({
+  formGroup: {
+    alignItems: 'center',
+    position: 'absolute',
+    height: '100%',
+    width: '100%',
+  },
+  form: {
+    width: '100%',
+  },
+  paper: {
+    marginTop: '10%',
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+  },
+  pageButtons: {
+    marginTop: '50%',
+  },
+  backButton: {
+    marginRight: theme.spacing(1),
+  },
+  instructions: {
+    marginTop: theme.spacing(1),
+    marginBottom: theme.spacing(1),
+  },
+  underlinedSubtitleText: {
+    textDecoration: 'underline',
+    textDecorationColor: '#53AA48',
+    fontWeight: 'bold',
+    marginBottom: '2%',
+
+  },
+  subtitleText: {
+    textAlign: 'left',
+    fontSize: '30px',
+    fontWeight: 'bold',
+    marginBottom: '7%',
+  },
+  regSubtitleText: {
+    textAlign: 'left',
+    paddingLeft: '10%',
+    fontSize: '30px',
+    marginBottom: '2%',
+  },
+  cenSubtitleText: {
+    textAlign: 'center',
+    fontSize: '30px',
+    fontWeight: 'bold',
+    marginBottom: '7%',
+  },
+  cenRegSubtitleText: {
+    textAlign: 'center',
+    paddingLeft: '10%',
+    fontSize: '30px',
+    marginBottom: '2%',
+  },
+  labelText: {
+    fontWeight: 'bold',
+  },
+  grayButton: {
+    backgroundColor: '#5e5e5e',
+    color: '#fff',
+    '&:hover': {
+      backgroundColor: '#5e5e5e',
+    },
+  },
+  blackButton: {
+    backgroundColor: '#373737',
+    color: '#fff',
+    '&:hover': {
+      backgroundColor: '#373737',
+    },
+    marginTop: '10%',
+  },
+  greenButton: {
+    backgroundColor: '#53AA48',
+    '&:hover': {
+      backgroundColor: '#53AA48',
+    },
+    marginTop: '10%',
+  },
+  submitButton: {
+    backgroundColor: '#53AA48',
+    '&:hover': {
+      backgroundColor: '#53AA48',
+    },
+    width: '50%',
+  },
+  textBox: {
+    width: '70%',
+  },
+  card: {
+    margin: 16,
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'space-between',
+  },
+  stepper: {
+    margin: 'auto',
+    width: '60%',
+  },
+  icon: {
+    color: 'red !important',
+  },
+});
+const ROLE_TITLES = ['buyer', 'nonprofit', 'seller'];
+const BUTTONS = [
+  {
+    id: 0,
+    title: 'Buyer',
+    image: Fruit3,
+    styling: useStyles.fruit3,
```

Comment:
Thats a good catch! The "styling: useSty..." stuff is actually redundant bc of the fruit: { ... } css right below that. The reason why I have the css in that js array tho instead of in the makeStyles above, is bc this makes it easier when iterating through the buttons in the for loop below. I just need to call bt.fruit instead of having an if, elseif, else clause.

As for the the names of the fruits being backwards, yeah that is annoying loll. I copied the naming convention used in the figma tho, (where the left most fruit corresponded to Fruit3, and so on).